src/util/wallet-utils/src/getAddress.ts

```diff
@@ -59,9 +62,9 @@ export function getWalletAddresses(
   }
 
   const stacks = getAddressByPurpose(response, AddressPurpose.Stacks);
-  if (!stacks) {
-    throw new Error("Stacks address not found");
-  }
+  // if (!stacks) {
+  //   throw new Error("Stacks address not found");
+  // }
```

Comment:
It also seems we are not using `taproot` around, should we comment out that check as well?

Reply:
good point!